v2/js/spa.js

```diff
@@ -160,18 +204,18 @@
     }
 
     SPA.prototype.toggleIdentities = function (action) {
+        const self = this;
         lpTag.identities = lpTag.identities || [];
         lpTag.sdes = lpTag.sdes || [];
         if (action === 'login') {
             lpTag.identities.push(getAuthData);
             lpTag.sdes.push({type: 'ctmrinfo', info: {customerId: 'lpTest' + userName}});
-
-            window.LPJsMethodName = function (callback) {
-                callback(userName);
-            };
-
-            window.LPGetAuthenticationToken = function (callback) {
-                callback(userName);
+            window.LPJsMethodName = window.LPGetAuthenticationToken = function (callback) {
```

Comment:
I'm not a big fan of setting values this way but it's not a huge deal

Reply:
Segregated and moved to a separate method